clusterloader2/cmd/clusterloader.go

```diff
@@ -39,7 +39,10 @@ var (
 func initClusterFlags() {
 	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "", "Path to the kubeconfig file")
 	// TODO(krzysied): query actual number of nodes with client (if not set).
-	pflag.IntVar(&clusterLoaderConfig.ClusterConfig.Nodes, "nodes", 0, "number of nodes")
+	pflag.IntVar(&clusterLoaderConfig.ClusterConfig.Nodes, "nodes", 0, "Number of nodes")
+	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.Provider, "provider", "", "Cluster provider")
+	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.MasterIP, "master-ip", "", "IP of the master node")
+	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.MasterName, "master-name", "", "Name of the master node")
```

Comment:
Why do you want this? This is implementation detail. What about HA clusters?

Reply:
I assume that most of the variables will be automatically set based on data form cluster client. However, if someone wants to set specific value, there should be a way to do this.

Reply:
So what's the point in exposing them now? Do we really need it now?
If not - let's not overgeneralize until it's really needed.

Reply:
So let me clarify my point: - I'm fine with making those part of ClusterLoader config.
But let's not expose them via flags until it's really needed.

Reply:
For example `master-ip` is used for sshing into the master node. Etcd metrics are using it to access the master. If master is not register AFAIK there is no way to calculate master ip - it needs to be provided.

Reply:
Even though in theory it's possible, we don't have such usecase for now.
So until it's really needed, let's avoid adding flags for such things.

Reply:
OK. Flags removed.